README.md

```diff
@@ -1,44 +1,129 @@
+
+The fork supports Python 3, increases test coverage, set up CI and linters, maintains releases to [PyPI](https://pypi.org/project/galgebra/#history), improves [docs](http://galgebra.readthedocs.io) and has many bug fixes.
+
+Features
+--------------------
+
+### Geometric Algebra
+
+- Arbitrary Vector Basis And Metric
+- Scalar, Vector, Bivector, Multivector, Pseudoscalar, Spinor, Blade
+- Basic Geometic Algebra Operations
+  - Sum Difference
+  - Geometric Product
+  - Outer And Inner Products
+  - Left And Right Contractions
+  - Reverse, Dual, Exponential
+  - Commutator
+  - Projection, Reflection, Rotation
+  - Reciprocal Frames
+- Inspecting Base Blade Representation
+- Symbolic Manipulations
+  - expand, factor, simplify, subs, trigsimp etc.
+
+Overloaded Python operators for basic GA operations:
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/basic_op.svg?sanitize=true)
+
+### Geometric Calculus
+
+- Geometric Derivative
+- Submanifolds
+- Linear Transformations
+- Differential Operators
+
+The various derivatives of a multivector function is accomplished by multiplying the gradient operator vector with the function:
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/grad.svg?sanitize=true)
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/grad_cmp.svg?sanitize=true)
+
+An example for getting `grad` and `rgrad`:
+
+```python
+(ex, ey, ez, grad, rgrad) = 
+  MV.setup('e*x|y|z', metric='[1,1,1]', coords=symbols('x y z'))
+```
 
-Installing Galgebra
+### Printing
+
+- Enhanced Console Printing
+- Latex Printing
+  - out-of-the-box support for Jupyter Notebook
+  - PDF generation and croping support if you have `pdflatex`/`pdfcrop` installed
+
+Getting Started
+---------------------
+
+After [Installing GAlgebra](#installing-galgebra), in a Jupyter Notebook
+
+```python
+from sympy import symbols
+from galgebra.ga import Ga
+
+from galgebra.printer import Format
+Format(Fmode = False, Dmode = True)
+
+st4coords = (t,x,y,z) = symbols('t x y z', real=True)
+st4 = Ga('e_t e_x e_y e_z',
+         g=[1,-1,-1,-1],
+         coords=st4coords)
+
+M = st4.mv('M','mv',f = True)
+
+M.grade(3).Fmt(3,r'\langle \mathbf{M} \rangle _3')
+```
+
+will output
+
+![](https://raw.githubusercontent.com/pygae/galgebra/nbsphinx/doc/images/st4_M3.svg?sanitize=true)
+
+NOTE: If you are from [sympy.galgebra](https://docs.sympy.org/0.7.6.1/modules/galgebra/) or [brombo/galgebra](https://github.com/brombo/galgebra), please check out [Migration Guide](#migration-guide).
+
+Installing GAlgebra
 ---------------------
 
 ### Dependencies
 
-- [Python](https://www.python.org/) 2.7 & 3
-  - tests pass under both Python 2.7 & 3.6
+- [Python](https://www.python.org/) 2.7 or 3
 - [SymPy](https://www.sympy.org)
 
-### Installing Galgebra From PyPI
+### Installing GAlgebra From PyPI
 
 ```bash
 pip install galgebra
 ```
 
-### Installing Galgebra From Source
+### Installing GAlgebra From Source
```

Comment:
A tiny suggestion (no strong feelings either way):

```suggestion
### Installing GAlgebra From Source Code
```

Reply:
"install from source" is quite frequently used and when there's code, it will has to be "install from the source code" which is a little bit verbose. So I'll prefer keeping it.

Reply:
Sure 👍